src/main/java/net/pms/util/OpenSubtitle.java

```diff
@@ -1380,10 +1208,8 @@ protected static String guessImdbIdByFileName(
 
 		URL url = login();
 		if (url == null) {
-			LOGGER.error(
-				"Couldn't guess IMDB ID for {} since OpenSubtitles login failed",
-				resource == null ? fileName : resource.getName()
-			);
+			LOGGER.error("Couldn't guess IMDB ID for {} since OpenSubtitles login failed",
+				resource == null ? fileName : resource.getName());
```

Comment:
```suggestion
			LOGGER.error(
				"Couldn't guess IMDB ID for {} since OpenSubtitles login failed",
				resource == null ? fileName : resource.getName()
			);
```